lib/tpm2_options.c

```diff
@@ -233,6 +233,29 @@ void tpm2_print_usage(const char *command, struct tpm2_options *tool_opts) {
     }
 }
 
+static TSS2_RC tcti_fake_receive (TSS2_TCTI_CONTEXT *tctiContext, size_t *response_size,
+        unsigned char *response_buffer, int32_t timeout) {
+	(void) tctiContext;
+	(void) response_size;
+	(void) response_buffer;
+	(void) timeout;
+
+	return TSS2_TCTI_RC_NOT_IMPLEMENTED;
```

Comment:
I would toss some error logs here, because if this ever gets invoked we want to know exactly why it failed simply by looking at logs.